docs/README.ingress-tls.md

```diff
@@ -0,0 +1,9 @@
+# Using Secrets Store CSI to Enable NGINX Ingress Controller with TLS
+
```

Comment:
Since we already have links that point to `sample/ingress-controller-tls/README.md`, how about we replace the content this?

Reply:
Makes sense! Kept the change minimal and updated `sample/ingress-controller-tls/README.md`.